lib/hyrax.rb

```diff
@@ -21,6 +21,9 @@
 require 'hyrax/name'
 require 'hyrax/valkyrie_can_can_adapter'
 require 'kaminari_route_prefix'
+require 'valkyrie/indexing_adapter'
```

Comment:
I don't know that these requires are necessary.  I was able to use the solr_adapter with just a change to indexing_adapter_initializer.rb.

Reply:
these are being moved from the initializer, here.

are you saying that you were able to drop them from initializer without replacing them? i wonder how this code is being loaded?

Reply:
No I did not remove them. If this was a move, then I was likely picking them up in their original location.